spec/config_spec.rb

```diff
@@ -236,5 +241,17 @@
       local = Pry::Config.from_hash({}, nil)
       expect(local['output']).to eq(nil)
     end
+
+    describe "Pry::Config::Lazy" do
+      specify "Pry::Config::Lazy objects are called when read" do
```

Comment:
What do you think about sticking to `it`? `specify` is long and we don't really use it that much. I don't think it's good practice to use it unless it's a one-liner. https://stackoverflow.com/questions/8484133/difference-between-an-it-block-and-a-specify-block-in-rspec

Reply:
i find always using `it` forces a limited vocabulary to describe a test.

Reply:
so i usually vary between the test, and choose whichever can describe the test best.

Reply:
That makes sense but it's really easy to convert this into the `it` form:

```
context "when given a Pry::Config::Lazy object" do
  it "invokes #call on it" do
```

Besides, this is nested within `#[]` examples but these tests don't call `#[]`, so it looks like it needs some better place.

Reply:
you're right, we should be using `#[]` directly and not doing that through `method_missing`.
agreed about the test description as well.
i will update sometime later, i'm back at work again so i won't have as much free time